.github/workflows/ghcr_publish.yml

```diff
@@ -0,0 +1,37 @@
+name: GHCR Publish
+
+# when a new release is created or new tag is pushed to master
+# tag should be in the format v1.0.0
+# regex to check the tag format
+on:
+  release:
+    types: [created]
+  push:
+    tags:
+        - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+    build:
+        uses: ./.github/workflows/rust_build.yml
+    publish:
+        runs-on: ubuntu-latest
+        needs: build
+        if: needs.build.result == 'success'
+        steps:
+            - name: Checkout the repo
+              uses: actions/checkout@v3
+            - name: Login to GitHub Container Registry
+              uses: docker/login-action@v1
+              with:
+                registry: ghcr.io
+                username: ${{ github.actor }}
+                password: ${{ secrets.GITHUB_TOKEN }}
+            - run: bin/ubuntu-setup.sh
+
+            - name: Build and publish scylla image with ver and latest tag
+              run: sh bin/ghcr-publish.sh  scylla_pg_monitor
+            
+            - name: Build and publish the pg-migration Docker image
```

Comment:
we need to publish db_migration not scylla_pg_core. Can be renamed to scylla_pg_migration.